### PR TITLE
The Code was formatted. Tests for the feed/entity endpoint with default and non-default values for the query parameters were added.

### PR DESCRIPTION
**Pull request by mpushnikA1QA**
The code was formatted. Tests for the feed/entity endpoint with default and non-default values for the query parameters were added.

### CODE REVIEW

File: src/test/steps/checkNextPageAndPreviousPageFields.js [hunk 1]
```diff
@@ -17,87 +20,182 @@ const FEED_VALUE = "feed";
...
+  case "feed":
+    return "feed";
+  case "search":
+    return "search";
   default:
-    expectChai.fail(`The condition [${conditional}]  should be "feed" or "search"`);
+    expectChai.fail(
+      `The condition [${conditional}]  should be "feed" or "search"`
+    );
     break;
```
Comment (danil-shamsiev) [inline src/test/steps/checkNextPageAndPreviousPageFields.js:199]: You can remove this break

File: src/test/steps/сheckSearchData.js [hunk 2]
```diff
@@ -5,94 +5,185 @@ const expectChai = chai.expect;
...
+        a[`${notContainsKey}`].toLowerCase()
+      );
+      expectChai(notContainsValueArrayByKey).all.to.not.include(
+        notContainsValue.toLowerCase(),
+        `The values for the key [${notContainsKey}]  should not contain the value - lowercase [${notContainsValue}]`
+      );
+      break;
+    default:
+      expectChai.fail(`This conditional [${conditional}] is not supported.`);
+      break;
```
Comment (danil-shamsiev) [inline src/test/steps/сheckSearchData.js:186]: You don't need break in default

**Review by danil-shamsiev — CHANGES_REQUESTED** [review]

File: src/test/steps/checkTheRequestToTheFeedEndpointWithAllParameters.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+const { ENTITIES_ENDPOINT, ENTITIES_ID } = require("../../constants/entities");
+const { SORT_FIELDS } = require("../../constants/apiSortFields");
+const { queryFields, commonFields } = require("../../testData/fields/fields");
+
+When(
+  /^Send a request to the feed\/'(.*)'\?fields&exclude&sort&order&pagesize endpoint with four items for fields and non-default values for the query parameters$/,
+  async function (entity) {
+    /*TODO: Now the logic check will be performed only for entities -drug, trial, investigator, organization, organizationHierarchy.
+    For the rest, you need to sort out the fields for sorting.*/
+    const fieldsCount = 4;
```
Comment (agromyko) [inline src/test/steps/checkTheRequestToTheFeedEndpointWithAllParameters.js:25]: please set this number as a parameter for the method as "endpoint with '4' items for fields" so we could change it if needed in the scenario

File: src/test/steps/checkNextPageAndPreviousPageFields.js (hunk 1 above)
Comment (mpushnikA1QA) [reply src/test/steps/checkNextPageAndPreviousPageFields.js:199]: Fxied

File: src/test/steps/сheckSearchData.js (hunk 2 above)
Comment (mpushnikA1QA) [reply src/test/steps/сheckSearchData.js:186]: Updated

File: src/test/steps/checkTheRequestToTheFeedEndpointWithAllParameters.js (hunk 3 above)
Comment (mpushnikA1QA) [reply src/test/steps/checkTheRequestToTheFeedEndpointWithAllParameters.js:25]: Updated

**Review by danil-shamsiev — APPROVED** [review]

**Pull request merged**